canonical/tempo-coordinator-k8s-operator
# added resolver directive

Fixes https://github.com/canonical/cos-lib/issues/94

This change tells Nginx to update the IP whenever it changes.
To test:
deploy COS-lite plus Tempo HA.
refresh tempo worker with any locally packed charm
verify you can ping the /ready endpoint on port 3200 soon (max 5sec) after the new worker service has reported ready.
